dlrover/python/common/constants.py

```diff
@@ -354,3 +354,10 @@ class ErrorMonitorConstants(object):
     ACTION_RDZV_TIMEOUT = "rendezvous_timeout"
     ACTION_TRAINING_START = "training_start"
     ACTION_RESTART_TRAINING = "restart_training"
+    ACTION_START_SAVE_SHARD = "start_save_shard"
```

Comment:
![image](https://github.com/user-attachments/assets/79aa684a-6aa0-4a79-be7f-bb1965549544)
Please ensure consistency in this type of naming (order of verbs and nouns) in the future. Do not change it this time, as it will affect downstream processes.